db.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+const DB_NAME = process.env.DB_NAME;
+const DB_PASS = process.env.DB_PASS;
+const DB_HOST = process.env.DB_HOST;
+dotenv.config();
+
+const dbURL = `mongodb+srv://haythem:12TamaraAdem1990@cluster0.u0opl.mongodb.net/task?retryWrites=true&w=majority`;
```

Comment:
you were supposed to use the environment variables and never use your password directly to the link. Go back to mongodb and change your password